gufe/executors/executor.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+from typing import Iterable, Union
+
+from .results import ResultStoreClient
+
+
+class Executor:
+    """Destination for `AlchemicalNetwork` submission and result requests via `Client`.
+    
+    The `Executor` exposes a RESTful API for incoming `Client` requests.
+    It also coordinates all activity between `ResultStore` and connected `Scheduler`s.
+
+    Submitted `AlchemicalNetwork`s and `Strategy` are applied to produce
+    `ProtocolDAG`s, which are made available for retrieval by connected `Scheduler`s.
+    As `Scheduler`s complete `ProtocolDAG`s, they return `ProtocolResult`s to the `Executor`.
+    The `Executor` deposits these in the `ResultStore` for long-term persistence.
+
+    Attributes
+    ----------
+    results : ResultStoreClient
+        Client interface to storage resource for all depositing and retrieving
+        all `ProtocolResult`s for submitted `AlchemicalNetwork`s.
+    state : str
+        URI for state store; this is a log-oriented DB that can be used to
+        recapitulate current `Executor` state, or any past state.
+    config : str
+        URI for Executor configuration.
+
+    """
+    def __init__(self,
+            results: ResultStoreClient,
+            state: str,
+            ):
+        ...
+
+
+    def export_state(datetime: datetime.datetime):
```

Comment:
Is this meant to be `-> datetime`? I'm not sure how datetime is an argument

Reply:
Fixed!